### ISSUE TITLE
[FR] Refresh in HA Android App

### ISSUE THREAD

**Issue by barthel-eu** [NemesisRE/kiosk-mode#300]
I use the weather forecast card in our control panel running on Android (Fire 10 tablet). That keeps "hanging", instead of daily and below hourly forecast, it falls back to twice hourly forecast. When I refresh the page, all is good. I do not want to have the header (takes space I can use better), but I would like to be able to refresh the page by i.e. pulling down (intuitive). Or have a button I can add into my panel to refresh. Any ideas?

And yes, it's obviously a bug in the weather forecast, but there may be other reasons why someone may want to refresh on a kiosk mode that doesn't show the header and such has no access to the "classic" refresh? So my feature request.

Workaround so far is to change something minor in the settings on my HA, then HA asks on the panel to refresh. But that's not "convenient" and not working with the (non-IT) family.

**Comment by elchininet** [comment]
Hi @barthel-eu,
That falls out of the scope of `kiosk-mode`. Please, consult [this thread](https://github.com/NemesisRE/kiosk-mode/issues/262) it can give you some ideas about how to achieve that.

**Comment by elchininet** [comment]
Another idea that comes to my mind to solve your use case is to use one of the plugins that I maintain. With [home-assistant-secret-taps](https://github.com/elchininet/home-assistant-secret-taps). You could have a "secret" tap sequence to show the header and hide it again when the refresh is performed.

**Comment by elchininet** [comment]
Another option is to use [browser-mod](https://github.com/thomasloven/hass-browser_mod):

https://community.home-assistant.io/t/refresh-page-button/351046/5

**Comment by elchininet** [comment]
Hi @barthel-eu,
If you want to give a try to `home-assistant-secret-taps`, I have just released a new version that allows you to execute `JavaScript` code (so you can add a `location.reload()` to a secret). Check [the last secret in the example in the repo](https://github.com/elchininet/home-assistant-secret-taps?tab=readme-ov-file#configuration-example).

**Comment by elchininet** [comment]
Hi @barthel-eu,
Let me know if you need support setting up some of the alternatives.
Closing this issue. 
Regards

**Comment by barthel-eu** [comment]
Just as feedback: Without JavaScript experience, I've worked it out using the "auto-reload-card" installed via HACS. Which I've set with a condition inside the control panel, that shows it only to me and set it on a 15 minute reload. So far that seems to work, though sometimes it did work for some days and then the error happened. Which I learned not to be uncommon to loose connection inside HA on unattended systems like a control panel... So yes, I think, IMHO it should be a kiosk-related issue.
